.github/update-published-charms-tests-workflow.py

```diff
@@ -0,0 +1,132 @@
+#! /usr/bin/env python
```

Comment:
It wasn't obvious to me that this script needed to be run manually 🙈

Reply:
What would you recommend here? I could add a section to `CONTRIBUTING.md` that explains that this exists and how to update it, or I could just add that in the docstring for the script itself. Or I guess it could be run automatically on a schedule, which is probably cleanest, but I wasn't sure if we were sold enough on this idea to do that straight off.

Reply:
Yeah, I think manual updates are a good start. We can always run this automatically later if desired.